docs/getting_started.rst

```diff
@@ -365,6 +365,7 @@ You can initialize the server using:
 
   kolibri manage migrate
 
+To use a PostgreSQL database, you would edit the `options.ini` file in `KOLIBRI_HOME`, setting options under the `[Database]` section to appropriate values. For the `DATABASE_ENGINE` option, use the value `postgres`.
```

Comment:
I do think that we may want to hold off on adding this kind of Postgres documentation until we are able to produce more thorough documentation like the following:

https://github.com/learningequality/kolibri/issues/6480#issuecomment-583299221

I suggest that Kolibri + psql documentation should link to OS-specific installation instructions and provide universal details for how to setup proper roles, databases, etc to use psql with Kolibri. 

If we mention PostgreSQL only briefly, then I think we should note to users that we recommend PostgreSQL only for those who already familiar with setting it up - in which case I think that this paragraph would be very helpful for someone trying to figure out how/where to configure their database.